atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -504,6 +504,8 @@ def select_and_start_cluster(self, platform):
             for cluster_info in possible_cluster_info:
                 ctx = retry_contexts[cluster_info.cluster.name]
                 try:
+                    self.log.info('Attempting to start build for platform %s on cluster %s',
```

Comment:
Do we need a log message for when the build was actually started? Right before `build_info.watch_logs()` ?